Content.Shared/Item/HeldSpeedModifierSystem.cs

```diff
@@ -29,7 +29,7 @@ private void OnGotUnequippedHand(Entity<HeldSpeedModifierComponent> ent, ref Got
         _movementSpeedModifier.RefreshMovementSpeedModifiers(args.User);
     }
 
-    private void OnRefreshMovementSpeedModifiers(EntityUid uid, HeldSpeedModifierComponent component, HeldRelayedEvent<RefreshMovementSpeedModifiersEvent> args)
+    public (float,float) SetMovementSpeedModifiers(EntityUid uid, HeldSpeedModifierComponent component)
     {
         var walkMod = component.WalkModifier;
```

Comment:
Code should never be setting this directly because it will cause a lot of bugs when refreshed modifiers are being bulldozed.

Reply:
It was not setting anything, I just gave it a bad name, Its now GetHeldMovementSpeedModifiers for ease of understanding. That function is just code I moved out of OnRefreshMovementSpeedModifiers so I could use it elsewhere.